src/backend/main/py.main/caliopen_main/objects/message.py

```diff
@@ -116,6 +127,150 @@ def create_draft(cls, user_id=None, **params):
             raise exc
         return message
 
+    def patch_draft(self, patch, **options):
+        """operations specific to draft, before applying generic patch method"""
+
+        self.get_db()
+        self.unmarshall_db()
+        if not self.is_draft:
+            return err.PatchUnprocessable(message="this message is not a draft")
+
+        try:
+            params = dict(patch)
+            params.pop("current_state")
+            message_param = NewMessage(params)
+            message_param.validate()
+            valid_draft = Message.validate_draft_consistency(self.user_id,
```

Comment:
why are you calling a classmethod instead of your object method. self is a Message instance, so validate_draft_consistency is an available method